src/Factory.php

```diff
@@ -25,13 +25,6 @@
  */
 abstract class Factory
 {
-    /**
-     * Memoization of normalized parameters.
-     *
-     * @internal
-     * @var Parameters|null
-     */
-    protected ?array $normalizedParameters = null;
```

Comment:
this is not needed anymore, because we're scheduling entities for insert from an "after instantiate" callback, so we can pass the attributes for there